ConsoleApp1/Menu.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace ConsoleApp1
+{
+    class Menu
+	{
+		public Menu menu = null;
+
+		private Menu()
+		{
+			if (menu == null)
+			{
+				votingSystem = new VotingSystem();
+            }
+		}
+		public static Menu CreateInstance()
+		{
+			return new Menu();
+		}
+
+        VotingSystem votingSystem;
```

Comment:
add access modifier and move to the top of the class (all fields should be declared there)